frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -120,8 +125,30 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
   const operatorNs = getNamespace(subscription[0]);
   const csvName = subscription?.[0]?.status?.installedCSV;
   const nodeName = obj.metadata.name;
+  const nodeNameByHostnameLabel = obj.metadata?.labels?.['kubernetes.io/hostname'];
   const nodeRole = getNodeRole(obj);
   const propName = `lvdr-${nodeName}`;
+  const [errorMessage, setError] = React.useState('');
+  const [inProgress, setProgress] = React.useState(false);
```

Comment:
```suggestion
  const [lvdRequestError, setError] = React.useState('');
  const [lvdRequestInProgress, setProgress] = React.useState(false);
```
Also move hooks to top of the component and with other hooks.